src/archanaNJun20/PrintNumberException.java

```diff
@@ -0,0 +1,27 @@
+package archanaNJun20;
+import java.util.InputMismatchException;
+import java.util.Scanner;
+public class PrintNumberException {
+	void sumOfDigits() {
+		int[] array = new int[3];
+		int counter = 0, sum = 0;
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Enter numbers:");
+		while(counter<=2) {
+			try {
+				array[counter] = scanner.nextInt();
+				sum += array[counter];
+				counter++;
+			}catch(InputMismatchException ie) {
+				System.out.println("Please enter integer numeber");
+				scanner.next();
+			}
+		}
+		System.out.println("Sum of the integer numbers : "+sum);
```

Comment:
the expected output is "Output : values are 10,30,40 and total is 80" is this
we have to show the valid numbers as well for which we calculated sum.

Reply:
code is updated as per suggestion 
![image](https://user-images.githubusercontent.com/66842290/87955450-6632d280-cacb-11ea-942b-1c875c75a863.png)